frontend/src/translations.json

```
@@ -189,7 +195,9 @@
       "please_correct_errors": "Proszę poprawić następujące błędy:",
       "drives": "Trasy",
       "unexpected_error": "Wystąpił nieoczekiwany błąd. Prosimy o kontakt z administratorem.",
-      "yes": "tak"
+      "yes": "tak",
+      "are_you_sure": "Jesteś pewien?",
```

```suggestion
      "are_you_sure": "Jesteś pewien/pewna?",
```
So we're hitting some limitations regarding localisation :P Though we don't store the gender anywhere it still would be nice to be more versatile in this regard, hence my suggestion.